common-rendering/src/editorialPalette.ts

```diff
@@ -108,6 +112,47 @@ const textStandfirstDark = ({ design }: ArticleFormat): Colour => {
 	}
 };
 
+const blogsGrayBackgroundPalette = (format: ArticleFormat): string => {
+	switch (format.theme) {
+		case ArticlePillar.News:
+			return news[400];
+		case ArticlePillar.Lifestyle:
+			return lifestyle[400];
+		case ArticlePillar.Sport:
+			return sport[400];
+		case ArticlePillar.Culture:
+			return culture[300];
+		case ArticlePillar.Opinion:
+			return opinion[300];
+		case ArticleSpecial.Labs:
+			return labs[300];
+		case ArticleSpecial.SpecialReport:
+			return specialReport[300];
+	}
+};
+
+const textStandfirstLink = (format: ArticleFormat): Colour => {
+
+	switch(format.design) {
+		case ArticleDesign.LiveBlog:
+			return neutral[100];
+		case ArticleDesign.DeadBlog:
+			return blogsGrayBackgroundPalette(format);
+		default: {
+			if(format.theme == ArticleSpecial.Labs)
+				return labs[300];
+
+			const { kicker, inverted } = getThemeStyles(format.theme);
```

Comment:
Can we specify these colour rules in this file instead? Unfortunately there are a couple of reasons we might want to avoid using `getThemeStyles`:

- We'd like to avoid importing from DCR and AR into common-rendering; the dependency tree should be the other way around, i.e. DCR and AR import _from_ common-rendering
- I think it would be best to replace and eventually remove the theme styles logic, and we'd like to replace it with the logic in _this_ file - so taking logic from there and putting it here would be 👍